package.json

```diff
@@ -21,6 +21,7 @@
   "license": "MIT",
   "description": "JavaScript SDK for Copilot Extensions",
   "dependencies": {
+    "@copilot-extensions/preview-sdk": "^4.0.3",
```

Comment:
did you add this dependency on itself intentionally? If yes, why?

Reply:
No, probably ran an npm install command at the wrong folder. Removed it!